projects/pgai/pgai/cli.py

```diff
@@ -250,6 +213,13 @@ def shutdown_handler(signum: int, _frame: Any):
     show_default=True,
     help="Exit after processing all available work (implies --exit-on-error).",
 )
+@click.option(
+    "-c",
+    "--concurrency",
+    type=click.IntRange(1),
+    default=None,
```

Comment:
Making it optional, since we are either either using the concurrency configured in the vectorizer config or its default (1).